### PR TITLE
flatpak: 1.14.2 -> 1.14.4

### PR DESCRIPTION
**Pull request by yVieta**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/flatpak/default.nix [hunk 1]
```diff
@@ -61,7 +61,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   src = fetchurl {
     url = "https://github.com/flatpak/flatpak/releases/download/${finalAttrs.version}/flatpak-${finalAttrs.version}.tar.xz";
-    sha256 = "sha256-ijTb0LZ8Q051mLmOxpCVPQRvDbJuSArq+0bXKuxxZ5k="; # Taken from https://github.com/flatpak/flatpak/releases/
+    sha256 = "8a34dbd0b67c434e7598b98ec690953d046f0db26e480aeafb46d72aec716799"; # Taken from https://github.com/flatpak/flatpak/releases/
```
Comment (wegank) [inline pkgs/development/libraries/flatpak/default.nix:64]:
```suggestion
    sha256 = "sha256-ijTb0LZ8Q051mLmOxpCVPQRvDbJuSArq+0bXKuxxZ5k=";
```

**Comment by alyssais** [comment]
Please follow the commit message guidelines in CONTRIBUTING.md.

**Review by wegank — CHANGES_REQUESTED** [review]
Please squash the commits and fix the commit message.

**Comment by Quanterm** [comment]
> Please squash the commits and fix the commit message.

I have squashed it

**Review by wegank — CHANGES_REQUESTED** [review]
`flatpak` is already at `1.14.4`, so the ~commit message~ title isn't right.

**Comment by wegank** [comment]
> > Please squash the commits and fix the commit message.
> 
> I have squashed it

You haven't.

File: pkgs/development/libraries/flatpak/default.nix [hunk 2]
```diff
@@ -61,7 +61,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   src = fetchurl {
     url = "https://github.com/flatpak/flatpak/releases/download/${finalAttrs.version}/flatpak-${finalAttrs.version}.tar.xz";
-    sha256 = "sha256-ijTb0LZ8Q051mLmOxpCVPQRvDbJuSArq+0bXKuxxZ5k="; # Taken from https://github.com/flatpak/flatpak/releases/
+    hash = "sha256-ijTb0LZ8Q051mLmOxpCVPQRvDbJuSArq+0bXKuxxZ5k=";
```
Comment (getchoo) [inline pkgs/development/libraries/flatpak/default.nix:64]: what is this change for exactly?

Comment (wegank) [reply pkgs/development/libraries/flatpak/default.nix:64]: The comment is outdated.

Comment (getchoo) [reply pkgs/development/libraries/flatpak/default.nix:64]: but why the change to `hash`?

Comment (wegank) [reply pkgs/development/libraries/flatpak/default.nix:64]: That's indeed unneeded.

**Pull request closed without merging**